ihe-interop/src/main/java/org/motechproject/ihe/interop/domain/HL7Recipient.java

```diff
@@ -9,12 +9,25 @@
 
     private String recipientUrl;
 
+    private String recipientUsername;
+
+    private String recipientPassword;
+
     public HL7Recipient() {
     }
 
     public HL7Recipient(String recipientName, String recipientUrl) {
         this.recipientName = recipientName;
```

Comment:
Make it call this(args) with proper parameters. It's a good practice to chain constructors, so they do not repeat the same assignemnts over and over again